### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by harshkotkar5557**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <nav class="navbar_std grey-bg">
+        <header class="d-flex justify-space-bw align-center">
+          <div class="brandName"><a href="../index.html">Play-Q</a></div>
+          <div id="hambar" class="hambar"><i class="fa fa-bars" aria-hidden="true"></i></div>
+           <ul class="navbarOptions d-flex d-flex justify-space-bw align-center">
+                <li><a href="https://github.com/harshkotkar5557" alt="#"><i class="fa fa-github" 
+                aria-hidden="true">&nbsp;</i></a></li>
+                 <li><a href="https://www.linkedin.com/in/haresh-kotkar-389a08139/" alt="#"><i 
+                 class="fa fa-linkedin-square" aria-hidden="true">&nbsp;</i></a> </li>
+                 <li><a href="https://twitter.com/KotkarHaresh" alt="#"><i class="fa fa-twitter-square" aria-hidden="true">&nbsp;</i></a></li>
```
Comment (vaibhav18matere) [inline index.html:24]: add `alt` text also , it improves performance in screen readers

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      </header>
+</nav>
+<section>
+        <div class="d-flex align-center justify-around p-2rem">
+            <img src="./images/undraw_game_day_ucx9.svg" alt="" srcset="" class="w-half">
+            <div class="d-flex flex-dr-col">
+                <h3 class="">
+                    Are you smart ? 
+                </h3>
+                <h2 class="p-1">Lets try check ..</h2>
```
Comment (vaibhav18matere) [inline index.html:35]: try to use more descriptive title and subtitle

File: index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    <div class="card_info">
+                        <h3>
+                            IPL 2022 team
+                        </h3>
+                    </div>
+                </div>
+                <button class="btn secondary w-full">Play Now</button>
+            </div>
+            <div class="card shadow p-zero">
+                <div class="" >
```
Comment (vaibhav18matere) [inline index.html:55]: avoid writing `class=" "` , if you are not using it

File: style.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+}
+.p-t-5{
+    padding-top: 5px;
+}
+.p-t-10{
+    padding-top: 10px;
+}
+.p-t-15{
+    padding-top: 15px;
+}
```
Comment (vaibhav18matere) [inline style.css:51]: use more descriptive class names

**Review by vaibhav18matere — APPROVED** [review]

File: style.css (hunk 4 above)
Comment (harshkotkar5557) [reply style.css:51]: Okay I will do that

**Review by aamrapaliwandhre — APPROVED** [review]
looks good. go ahead.

**Pull request merged**